### ISSUE TITLE
Missed syntax error on empty domain

### ISSUE THREAD

**Issue by scripthunter7** [AdguardTeam/AGLint#143]
The parser throws an error on the empty domain in the domain list:
```adblock
example.com,,example.net##.selector
```

but does not detect if the last domain is missing. An error should be thrown on the following rule:

```adblock
example.com,example.net,##.selector
```

_Note:_ this case also works well, only one comma is problematic
```adblock
example.com,example.net,,##.selector
```

Reported by @Alex-302
